.github/workflows/androidTest.yml

```diff
@@ -0,0 +1,12 @@
+#Workflow name
+name: Automated Andriod Unit Tests
```

Comment:
Ik ga de merge request niet nu tegenhouden voor een spelfoutje 😉 
Fix het maar gewoon in je volgende PR.